docs/DeveloperGuide.md

```diff
@@ -486,367 +498,250 @@ Hence, any modification to an employee after SudoHR is initialized from storage
-| `* *`    | HR personnel                           | hide private contact details                                   | minimize chance of someone else seeing them by accident                    |
-| `* * *`  | HR personnel                           | add an employee’s leave to SudoHR                              | ensure consolidation of information                                        |
-| `* * *`  | HR personnel                           | remove an employee’s leave for SudoHR                          | ensure consolidation of information                                        |
-| `* * *`  | HR personnel                           | view all leaves an employee has applied for                    | access an employee's availability easily                                   |
-| `* * *`  | HR personnel                           | view all employees on leave today                              | know today's headcount                                                     |
-| `* * *`  | HR personnel                           | view all leaves applied for a given day                        | better plan company events                                                 |
-| `* * *`  | HR personnel                           | view all leaves applied for a given day for a given department | better plan depeartment events                                             |
-| `* * *`  | HR personnel                           | add a department                                               | ensure consolidation of information when a new department is formed        |
-| `* * *`  | HR personnel                           | edit a department                                              | ensure consolidation of information when a department's detail is changed  |
-| `* * *`  | HR personnel                           | delete a department                                            | ensure consolidation of information when a department is disbanded         |
-| `* * *`  | HR personnel                           | find a department by name                                      | locate details of departments without having to go through the entire list |
-| `* * *`  | HR personnel                           | add an employee to a department                                | ensure consolidation of information when a department has a new employee   |
-| `* * *`  | HR personnel                           | remove an employee from a department                           | ensure consolidation of information when an employee leaves a department   |
-| `* * *`  | HR personnel                           | list all departments an employee is in                         |                                                                            |
-| `* * *`  | HR personnel                           | list all employees in a department                             | view manpower size of a department                                         |
-
-
-*{More to be added}*
```

Comment:
It may be better to adjust priority for commands that we have implemented to be higher and the rest 2 * and below

Reply:
I did not touch the user stories part, I think we should do this in a separate issue.